server/bin/pbench-sync-satellite.sh

```diff
@@ -60,7 +60,7 @@ mkdir -p ${logdir} || log_exit "Failed to create ${logdir}"
 
 function do_remote_sat_state_change {
     local status
-    pbench-remote-satellite-state-change ${satellite_config} ${state_change_log} > ${logdir}/mv.log 2>&1
+    ssh ${remote_host} "${remote_opt}/bin/pbench-satellite-state-change ${remote_archive}" < ${state_change_log} > ${logdir}/mv.log 2>&1
```

Comment:
Not sure what tense to use here.

As implemented this code is fragile, for sure.  The choice we made to implement it this way was expedient at best for two people trying to support the agent code and implement a solution for the large shift in use cases that came about quickly in the cloud.

I'd like to change it, but we can't since the `pbench-satellite-state-change` in this branch, after the `0.69.10-server` tag, is not deployed on any satellite, since `pbench-dispatch` now only works with `pbench-results-push` to a `main` branch derived server sharing the file system with where this code is deployed.

If any of that makes sense.

Reply:
See also PR #3330.